server/tests/backends/fixtures/replace/with_variables.yaml

```diff
@@ -0,0 +1,115 @@
+---
+exclude:
+- mongo
+- pandas
+- snowflake
+step:
+  pipeline:
```

Comment:
Could you please add a `select` step to have case to have only the columns we're interested in ? Could you also add a replace case on "iple" to ensure "Triple" does not get modified ?

Reply:
`Ale` case is already here for that ;)

Reply:
Not really, since "India Pale Ale"  is a superset of the match (both contain "Ale"). With the previous behaviour, results could vary depending on the order of the replaces :man_shrugging: To completely validate that this is not the case anymore, we need a completely different match

Reply:
There is a second kind which contains Ale : 
`beer_kind: Belge blonde forte & Golden Ale`
This one isn't replaced

Reply:
ah true, nevermind